lib/User/Backend.php

```diff
@@ -147,6 +154,11 @@ public function getCurrentUserId() {
 			return '';
 		}
 
+		if ($this->providerService->getSetting($provider->getId(), ProviderService::SETTING_CHECK_BEARER, '1') !== '1') {
```

Comment:
```suggestion
		if ($this->providerService->getSetting($provider->getId(), ProviderService::SETTING_CHECK_BEARER, '1') !== '1') {
```

I'd vote for rather making it turned off by default, what do you think?

Reply:
Yes right, let's make it off by default.